src/main/java/com/exasol/sql/ddl/CreateTableColumns.java

```diff
@@ -0,0 +1,39 @@
+package com.exasol.sql.ddl;
+
+import com.exasol.datatype.DataType;
+import com.exasol.sql.AbstractFragment;
+import com.exasol.sql.Column;
+import com.exasol.sql.SqlStatement;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CreateTableColumns extends AbstractFragment implements CreateTableFragment {
```

Comment:
I think we can rename this to `ColumsDefinition` since that can be used in other contexts too. UDF definitions for example. See User Guide 6.1, Section 2.2. under "CREATE SCRIPT".

Reply:
done